examples/03_hybrid/src/App.tsx

```diff
@@ -1,6 +1,7 @@
 import { WritableAtom, atom, useAtom } from 'jotai';
 import { ScopeProvider } from 'jotai-scope';
 import { useHydrateAtoms } from 'jotai/react/utils';
+import { INTERNAL_InferAtomTuples } from 'jotai/react/utils/useHydrateAtoms';
```

Comment:
@dai-shi Here, since when jotai is bundled for release, utils is bundled as a single file, the node resolver cannot resolve `jotai/react/utils/useHydrateAtoms`. However, `jotai/react/utils/useHydrateAtoms.d.ts` exists and this type is not exported by `utils.d.ts`, so typescript has to use this path. Therefore, we need the typescript resolver for eslint-plugin-export